src/ossia-max/src/ossia-max.hpp

```diff
@@ -168,78 +183,62 @@ class ossia_max
 
   typedef std::map<t_object*, root_descriptor> RootMap;
 
+  struct patcher_descriptor{
+    ossia::safe_set<parameter*> parameters{};
+    ossia::safe_set<remote*>    remotes{};
+    ossia::safe_set<attribute*> attributes{};
+    model*     model{};
+    view*      view{};
+    device*    device{};
+    client*    client{};
+
+    t_object* parent_patcher;
+    ossia::safe_set<t_object*> subpatchers;
+
+    bool loadbanged; // true if patcher have been loadbanged already
```

Comment:
I think it should be initialized to false for safety

Reply:
done